pkg/network/ovn_kubernetes.go

```diff
@@ -239,6 +243,11 @@ func isOVNKubernetesChangeSafe(prev, next *operv1.NetworkSpec) []error {
 			errs = append(errs, errors.Errorf("cannot edit IPsec configuration at runtime"))
 		}
 	}
+	if pn.PolicyAuditConfig != nil && nn.PolicyAuditConfig != nil {
+		if !reflect.DeepEqual(pn.IPsecConfig, nn.IPsecConfig) {
```

Comment:
why not? This seems like a perfectly reasonable thing to be able to change?

Reply:
I agree! However this is what I saw in the Docs that made me make it changesafe 

`You can only change the configuration for your default CNI network provider during cluster installation.` 

--> https://docs.openshift.com/container-platform/4.7/networking/cluster-network-operator.html#nw-operator-configuration-parameters-for-ovn-sdn_cluster-network-operator

Reply:
I can do whatever you want here though I think it should be configurable at runtime as well

Reply:
Yes, make it reconfigurable (and update the docs, please :).